COPYING

```diff
@@ -1,345 +1,13 @@
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License along
-    with this program; if not, write to the Free Software Foundation, Inc.,
-    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-
-Also add information on how to contact you by electronic and paper mail.
-
-If the program is interactive, make it output a short notice like this
-when it starts in an interactive mode:
-
-    Gnomovision version 69, Copyright (C) year name of author
-    Gnomovision comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
-    This is free software, and you are welcome to redistribute it
-    under certain conditions; type `show c' for details.
-
-The hypothetical commands `show w' and `show c' should show the appropriate
-parts of the General Public License.  Of course, the commands you use may
-be called something other than `show w' and `show c'; they could even be
-mouse-clicks or menu items--whatever suits your program.
-
-You should also get your employer (if you work as a programmer) or your
-school, if any, to sign a "copyright disclaimer" for the program, if
-necessary.  Here is a sample; alter the names:
-
-  Yoyodyne, Inc., hereby disclaims all copyright interest in the program
-  `Gnomovision' (which makes passes at compilers) written by James Hacker.
-
-  <signature of Ty Coon>, 1 April 1989
-  Ty Coon, President of Vice
-
-This General Public License does not permit incorporating your program into
-proprietary programs.  If your program is a subroutine library, you may
-consider it more useful to permit linking proprietary applications with the
-library.  If this is what you want to do, use the GNU Lesser General
-Public License instead of this License.
+Except where noted otherwise in the file itself, the source code for all
+Pacemaker programs is licensed under version 2 or later of the GNU General
+Public License (GPLv2+), its headers and libraries under version 2.1 or
+later of the less restrictive GNU Lesser General Public License (LGPLv2.1+),
+its documentation under version 4.0 or later of the Creative Commons
+Attribution-ShareAlike International Public License (CC-BY-SA v4.0+),
+and its init scripts under the Revised BSD license.
+
+The text of these licenses are provided in the "licenses" subdirectory.
+
+If you find any deviations from this policy, or wish to inquire about alternate
+licensing arrangements, please e-mail pacemaker@oss.clusterlabs.org.
+Licensing issues are further discussed at: http://clusterlabs.org/wiki/License
```

Comment:
I plan on going with: "Licensing issues are further discussed on the ClusterLabs wiki (at http://clusterlabs.org/wiki/License)."

I don't like to end URLs with a bare period because viewers that auto-link URLs often mistakenly include the period in the link. They're usually more intelligent about parentheses, and always good with bare EOL.